.gitignore

```diff
@@ -4,3 +4,6 @@
 /.settings/
 /.cproject
 /.project
+/.idea/
+/Testing/
```

Comment:
Do not ignore personal directories like this. Ignoring `.idea` is acceptable since it is created by a commonly used IDE.